tests/scripts/parse_profile_logs.py

```diff
@@ -351,10 +350,6 @@ def clean_ops(ops):
                 row = parse_layernorm(key, val)
                 add_to_list(row, cleaned_ops["layernorm"])
                 # cleaned_ops['layernorm'].append(row)
-            elif "GroupNorm" in val[0]["NAME"]:
-                row = parse_groupnorm(key, val)
-                add_to_list(row, cleaned_ops["groupnorm"])
-                # cleaned_ops['groupnorm'].append(row)
```

Comment:
what a weird code. any idea what it did?

Reply:
not really sure, it is part of the infra used to test the "non-working groupnorm" so I removed it